ResourceStatusSystem/Agent/CacheFeederAgent.py

```diff
@@ -49,8 +49,6 @@ def initialize(self):
     """ Define the commands to be executed, and instantiate the clients that will be used.
     """
 
-    self.am_setOption('shifterProxy', 'DataManager')
```

Comment:
I believe this agent also gets the accounting information from the storage. In that case, you must have a proxy

Reply:
This call can be replaced by a relevant configuration option, so it is not required, is it ?

Reply:
Sure, but then it should be:
* In the doc
* not for a patch release

Reply:
I could re-base it wrt v7r0, would it be a better approach ?